### PR TITLE
Blog app setup and controllers

### PR DESCRIPTION
**Pull request by Mucyosoda**
Create a new app. Give it a meaningful name.
Initialize the project with Git.
Important: do NOT use rails to generate scaffold commands.

### CODE REVIEW

**Review by t-yanick — CHANGES_REQUESTED** [review]
Hi @Mucyosoda, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments below and  under the review._
### I will leave the comment here, since your `Readme file` is not accessible for this PR ###
- [x] Please kindly update your `readme file` to reflect the changes for this milestone as this will make your `readme file` more informative and professional. Checkout this [template](https://github.com/microverseinc/readme-template) and adapt it to your project. 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -1,24 +1,44 @@
...
-* How to run the test suite
-
-* Services (job queues, cache servers, search engines, etc.)
-
-* Deployment instructions
-
-* ...
+My Blog App
+Setup blog app website ou will create a fully functional website that will show the list of posts and empower readers to interact with them by adding comments and liking posts.
+
```
Comment (julie-ify) [inline README.md]: - [ ] Kindly create your README using this [template](https://github.com/microverseinc/readme-template/blob/master/README.md). Open the template and click on `raw` by the write side and then copy the README template to your own README and finally personalize it with the data you have now. Kindly ensure that the headlines are shown. See how your readme looks below; kindly make it look like the template.
chrome_sQsBnGmSJd

**Review by julie-ify — CHANGES_REQUESTED** [review]
Hi @Mucyosoda, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Mucyosoda) [reply README.md]: Thank you

**Review by elisha2kyakpo1 — APPROVED** [review]
Hi @Mucyosoda, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

### To improve
- The getting started section of a readme file should have all the commands required to run the project on a users local machine, from cloning to running the server.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**